graylog2-server/src/main/java/org/graylog2/plugin/configuration/ConfigurationRequest.java

```diff
@@ -150,6 +158,12 @@ public Configuration filter(Configuration config) {
                     }
                     break;
                 case TextField.FIELD_TYPE:
+                    break;
```

Comment:
Same as above, the fallthrough for the `DropdownField` implementation needs to stay for the `TextField`.